jobs/e2e_node/containerd/containerd-master/cgroupv2/env-cgroupv2

```diff
@@ -7,3 +7,5 @@ CONTAINERD_EXTRA_RUNTIME_HANDLER: 'test-handler'
 CONTAINERD_EXTRA_RUNTIME_OPTIONS: |
   BinaryName = "/home/containerd/usr/local/sbin/runc"
 CONTAINERD_CGROUPV2: 'true'
+CONTAINERD_SYSTEMD_CGROUP: 'true'
```

Comment:
Is it intentional to migrate `ci-cos-cgroupv2-containerd-node-e2e` to systemd too?

Reply:
It was already running using systemd, previously we were controlling it from containerd script now from here
https://github.com/containerd/containerd/pull/5884

Reply:
👍